docs/content/Getting-Started/quickstart.md

```diff
@@ -5,6 +5,15 @@ linkTitle: "KubeStellar Quickstart Guide"
 
 <img width="500px" src="../../KubeStellar with Logo.png" title="KubeStellar">
 
+
+## Demo Video
+
+Watch this video to see step-by-step demo of KubeStellar running and follow the instrunctions below to get your own KubeStellar quickly started.
```

Comment:
suggestion:

Watch this video to see a step-by-step demo of KubeStellar running and follow the instructions below to get your own KubeStellar started quickly.